google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/it/ITGcsNio.java

```diff
@@ -620,6 +667,32 @@ private String randomSuffix() {
     return "-" + rnd.nextInt(99999);
   }
 
+  private static class postTraversalWalker extends SimpleFileVisitor<Path> {
```

Comment:
This is a class so `PostTraversalWalker`?

Reply:
You're right, I fixed it now.